client/e2e/pages/onboarding/home/page.spec.ts

```diff
@@ -1,18 +1,206 @@
+        "efb76d57-88b7-4eb6-9f26-ec12b49c14c1",
+        "bceid_business_name",
+      ],
+    };
+    // ▶️ Execute the query
+    await pool.query(query);
+
+    // Upsert an User Operator record
+    query = {
+      text: `
+    INSERT INTO erc.user_operator (user_id, role, status, operator_id)
+    VALUES ($1,  $2, $3, $4)
+    ON CONFLICT (user_id, operator_id)
+    DO UPDATE SET role = EXCLUDED.role, status = EXCLUDED.status;
+  `,
+      values: [
+        process.env.E2E_INDUSTRY_USER_ADMIN_GUID as string,
+        "admin",
+        "Approved",
+        2,
+      ],
+    };
+    // ▶️ Execute the query
+    await pool.query(query);
+
+    // 👤 industry_user: bc-cas-dev-secondary
+    // Upsert a User record
+    query = {
+      text: `
+         INSERT INTO erc.user (user_guid, app_role_id, first_name, last_name , position_title , email, phone_number, business_guid, bceid_business_name )
+    VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)
+    ON CONFLICT (user_guid)
+    DO UPDATE SET app_role_id = EXCLUDED.app_role_id
+  `,
+      values: [
+        process.env.E2E_INDUSTRY_USER_GUID as string,
+        "industry_user",
+        "Cas SECONDARY",
+        "Bcgov",
+        "ADMINISTRATOR",
+        "email@email.com",
+        "+16044015432",
+        "efb76d57-88b7-4eb6-9f26-ec12b49c14c1",
+        "bceid_business_name",
+      ],
+    };
+    // ▶️ Execute the query
+    await pool.query(query);
+
+    // 👤 new user: bc-cas-dev-three
+    // Delete User record
+    query = {
+      text: "DELETE FROM erc.user WHERE user_guid = $1",
+      values: [process.env.NEW_USER_GUID as string],
+    };
+    // ▶️ Execute the deletion query
+    await pool.query(query);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error("❌ Error in Db setup for login roles:", error);
+    throw error;
+  }
+});
+
 test.describe("Test Page - Home", () => {
-  test("Test Login", async ({ page }) => {
-    await page.goto(url);
-    await expect(page.url().toLocaleLowerCase()).toContain("/home");
-    await expect(page.getByText("Welcome")).toBeVisible();
+  test("Test Route", async ({ page }) => {
+    const homePage = new HomePOM(page);
+    await homePage.route();
+    // 🔍 Assert correct url
+    homePage.urlIsCorrect();
+  });
+
+  test("Test Selfie", async ({ page }, testInfo) => {
+    const homePage = new HomePOM(page);
+    await homePage.route();
+    // 🔍 Assert that the content is correct
+    // Note: When you run snapshot for the first time the test runner will Error: A snapshot doesn't exist...
+    // that's because there was no golden file...
+    // but, this method took a bunch of screenshots until two consecutive screenshots matched, and saved the last screenshot to file system...
+    // it is now ready to be added to the repository and expected to pass test
+    await expect(page).toHaveScreenshot();
+    // 👀 Attach the screenshot to the report
+    const screenshot = await page.screenshot();
```

Comment:
Is this visual regression testing? We have a [card about doing that with happo](https://github.com/orgs/bcgov/projects/122/views/2?filterQuery=happo&pane=issue&itemId=48611123). But since this way is already implemented, let's see how it works for us and maybe we can skip happo instead